src/renderer/react/priceBox/components/price-box.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { CryptoPriceBox } from 'selfkey-ui';
+
+class PriceBox extends React.Component {
```

Comment:
What is this component for?

Reply:
@dfcowell good point, we can remove it and move the CryptoPrixeBox directly to the container. cheers